client/remote.cpp

```diff
@@ -586,6 +586,12 @@ static int build_remote_int(CompileJob &job, UseCSMsg *usecs, MsgChannel *local_
                 throw remote_error(102, "Error 102 - command needs stdout/stderr workaround, recompiling locally");
             }
 
+            if (crmsg->err.find("file not found") != string::npos) {
+                delete crmsg;
+                log_info() << "remote is missing file, recompiling locally" << endl;
+                throw remote_error(103, "Error 104 - remote is missing file, recompiling locally");
```

Comment:
Should this have said "Error 103"? :)

Reply:
there are 2 hard problems in computing:
- caching
- naming
- off by one errors